index.js

```diff
@@ -183,10 +183,13 @@
          */
         render: function() {
           var passedProps = this.props;
-          var props = { ref: 'instance' };
+          var props = {};
           Object.keys(this.props).forEach(function(key) {
             props[key] = passedProps[key];
           });
+          props.ref = 'instance';
+          props.disableOnClickOutside = this.disableOnClickOutside;
+          props.enableOnClickOutside = this.enableOnClickOutside;
```

Comment:
Alternatively, set this up above, in `props = {}`. I think it's okay to have them overridden if the user chooses to do so.